### PR TITLE
added observedLocation, put other Location refs back

### PR DESCRIPTION
**Pull request by fosterlynn**
Added observedLocation to EconomicEvent, per discussion https://github.com/valueflows/valueflows/issues/486.  Also put vf:Location back for now.  We can swap in another vocab for this if we want.  I'll add a few property references from other vocabs, unless you get there first @elf-pavlik .

### CODE REVIEW

**Comment by elf-pavlik** [comment]
We started moving away from creating different properties for different types of flows. Can we also here just use same property that would reference a place from Intent, Commitment and Event? Maybe simple `vf:atLocation` ?

**Comment by fosterlynn** [comment]
>Maybe simple vf:atLocation ?

Good idea.  Changed and pushed.

File: release-doc-in-process/all_vf.TTL [hunk 1]
```diff
@@ -367,17 +367,24 @@ vf:definedQuantity a        owl:ObjectProperty ;
...
         rdfs:label          "primary location" ;
         rdfs:domain         foaf:Agent ;
-        #rdfs:range          vf:Location ;
+        rdfs:range          vf:Location ;
         vs:term_status      "unstable" ;
         rdfs:comment        "The main place an agent is located, often an address where activities occur and mail can be sent. This is usually a mappable geographic location.  It also could be a website address, as in the case of agents who have no physical location." .
 
+vf:atLocation a             owl:ObjectProperty ;
+        rdfs:label          "at location" ;
+        rdfs:domain         [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent) ] ;  ;
```
Comment (elf-pavlik) [inline release-doc-in-process/all_vf.TTL]: somehow you have two semicolons at the end of the line

Comment (fosterlynn) [reply release-doc-in-process/all_vf.TTL]: aargh, thanks for catching!  fixed

**Pull request merged**